README.rst

```diff
@@ -0,0 +1,179 @@
+===============
+xblocks-contrib
+===============
+
+This repository is the new home for XBlocks, a core component of the Open edX ecosystem.
+This project involves the extraction of XBlocks from the edx-platform.
+
+Project Overview
+=======================
+
+XBlocks are modular components that enable rich interactive learning experiences in Open edX courses.
+Historically, the XBlock code was tightly coupled with the edx-platform, making it challenging to manage and extend.
+By extracting XBlocks into this dedicated repository, we can reduce the complexity of the edx-platform, making it more maintainable and scalable.
+
+Xblocks being moved here are::
+* poll_question
+* word_cloud
+* annotatable
+* lti
+* html
+* discussion
+* problem
+* video
+* videoalpha
```

Comment:
```suggestion
XBlocks Being Moved Here
************************

These are the XBlocks being moved here, and each of their statuses:

* ``poll_question`` -- Placeholder
* ``word_cloud`` -- Placeholder
* ``annotatable`` -- Placeholder
* ``lti`` -- Placeholder
* ``html`` -- Placeholder
* ``discussion`` -- Placeholder
* ``problem`` -- Placeholder
* ``video`` -- Placeholder

The possible XBlock statuses are:

* Placeholder: It's just a cookiecutter thumbs-up block.
* In Development: We're building and testing this block.
* Ready to Use: You can try this on your site using the Waffle flag.
* Done The built-in block has been removed. The setup.py entrypoint has been removed from edx-platform and added to xblock-contrib.
```

thanks for the README. a few tweaks on this list:
* you need a blank line before the list or the rst won't render right
* videoalpha is just an alias to video, so we can leave it off the list
* let's make this a heading and include extraction statuses, so we can use this list as a way to keep folks updated on which blocks are ready to use and which aren't